back-end/e-learning/routes/api.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Route;
+use App\Http\Controllers\UserController;
+/*
+|--------------------------------------------------------------------------
+| API Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you can register API routes for your application. These
+| routes are loaded by the RouteServiceProvider within a group which
+| is assigned the "api" middleware group. Enjoy building your API!
+|
+*/
+
+Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
+  return $request->user();
+});
+
+Route::group(['middleware' => 'auth:sanctum'], function(){
+  Route::get('me',[UserController::class,'me']);
+  Route::post('logout',[UserController::class,'logout']);
```

Comment:
You forgot the space after comma..

```suggestion
  Route::get('me', [UserController::class, 'me']);
  Route::post('logout', [UserController::class, 'logout']);
```